Lab/src/main/java/agh/ics/oop/MapBoundary.java

```diff
@@ -0,0 +1,13 @@
+package agh.ics.oop;
+import java.util.PriorityQueue;
+public class MapBoundary implements IPositionChangeObserver {
+    PriorityQueue<Integer> xAxis = new PriorityQueue<>();
+    PriorityQueue<Integer> yAxis = new PriorityQueue<>();
+    @Override
+    public void positionChanged(Vector2d oldPosition, Vector2d newPosition) {
```

Comment:
to jest ok, ale brakuje jeszcze metod do wstawiania w momencie gdy położymy zwierzę/trawę na mapie - teraz tylko reagujemy na zmiany pozycji.